src/main/java/dukeclass/Storage.java

```diff
@@ -0,0 +1,138 @@
+package dukeclass;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file
+ */
+public class Storage {
+
+    /**
+     * Create the file and directory for storage if it does not exist yet.
+     *
+     * @return File for storage of TaskList.
+     */
+    public static File createFile() {
+        File d = new File("data");
```

Comment:
Perhaps a more descriptive variable name could be used since the scope of this variable is not that small? Same for some of the other one letter variables.